### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by le2Ks**

### CODE REVIEW

**Comment by koh-gt** [comment]
Hi Tyler Anderson

File: src/chainparams.cpp [hunk 1]
```diff
@@ -92,10 +92,10 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT; 
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000000000000000000200020");
+        consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000000000000000136e15425");
```
Comment (koh-gt) [inline src/chainparams.cpp:95]: A bit high given there are no specialised GPU/FPGA miners available in addition to small number of miners.

File: src/chainparams.cpp [hunk 2]
```diff
@@ -67,7 +67,7 @@ class CMainParams : public CChainParams {
         consensus.nSubsidyHalvingInterval = 301107;
         consensus.BIP16Height = 0; 
         consensus.BIP34Height = 0;
-        consensus.BIP34Hash = uint256();
+        consensus.BIP34Hash = uint256S("0x46ca17415c18e43f5292034ebf9bbd10de80a61fc6dc17180e6609f33d3b48f3");
```
Comment (koh-gt) [inline src/chainparams.cpp:70]: I believe this is the hash of the block that corresponds to when BIP34 is activated, or block 4264.

File: src/chainparams.cpp [hunk 3]
```diff
@@ -121,8 +121,11 @@ class CMainParams : public CChainParams {
         // This is fine at runtime as we'll fall back to using them as a oneshot if they don't support the
         // service bits we want, but we should get them updated to support all service bits wanted by any
         // release ASAP to avoid it where possible.
-        vSeeds.emplace_back("118.189.201.104");
+
         vSeeds.emplace_back("node2.walletbuilders.com");
+        vSeeds.emplace_back("118.189.201.104");
+        vSeeds.emplace_back("seed01.altcoinbuilders.com");
+        vSeeds.emplace_back("seed02.altcoinbuilders.com");
```
Comment (koh-gt) [inline src/chainparams.cpp:128]: Are these nodes active?

File: src/chainparams.cpp [hunk 4]
```diff
@@ -142,17 +145,24 @@ class CMainParams : public CChainParams {
...
             }
         };
 
         chainTxData = ChainTxData{
-            /* nTime    */ 1669136135,
-            /* nTxCount */ 0,
-            /* dTxRate  */ 0
+            /* nTime    */ 1669559791,
+            /* nTxCount */ 4073,
+            /* dTxRate  */ 0.00969754
```
Comment (koh-gt) [inline src/chainparams.cpp:161]: checkpoint

**Review by koh-gt — APPROVED** [review]

File: src/chainparams.cpp (hunk 1 above)
Comment (koh-gt) [reply src/chainparams.cpp:95]: > A bit high given there are no specialised GPU/FPGA miners available in addition to small number of miners.

No. That is the total amount of work needed to produce the chain
0x0000000000000000000000000000000000000000000000000000000136e15425 
is 5215704101 hashes, which given an estimated 3 days of work is an average of 20kH/s. The current hashrate is 3MH/s likely from a midrange GPU or CPU cluster. which can clear this value in just 30 minutes.

Comment (le2Ks) [reply src/chainparams.cpp:95]: > > A bit high given there are no specialised GPU/FPGA miners available in addition to small number of miners.
> 
> No. That is the total amount of work needed to produce the chain 0x0000000000000000000000000000000000000000000000000000000136e15425 is 5215704101 hashes, which given an estimated 3 days of work is an average of 20kH/s. The current hashrate is 3MH/s likely from a midrange GPU or CPU cluster. which can clear this value in just 30 minutes.

Hello, `0000000000000000000000000000000000000000000000000000000136e15425`  is the chainwork from block height 4072 - used for the last checkpoint.

File: src/chainparams.cpp (hunk 2 above)
Comment (le2Ks) [reply src/chainparams.cpp:70]: > I believe this is the hash of the block that corresponds to when BIP34 is activated, or block 4264.

Yes correct, it is the block hash of height 0 (the genesis) when BIP34 has been activated.

File: src/chainparams.cpp (hunk 3 above)
Comment (le2Ks) [reply src/chainparams.cpp:128]: > Are these nodes active?

Yes active at the moment and for a long period of time.

File: src/chainparams.cpp (hunk 4 above)
Comment (le2Ks) [reply src/chainparams.cpp:161]: > checkpoint

Yes correct, these are made from the last checkpoints, at block height 4072

**Comment by le2Ks** [comment]
> Hi Tyler Anderson

Hello! Is there a discord or telegram group for your crypto project?

**Comment by koh-gt** [comment]
> > Hi Tyler Anderson
> 
> Hello! Is there a discord or telegram group for your crypto project?

I will create one. t.me/ferrite_core

**Pull request merged**